oidc-authentication.md

```diff
@@ -289,19 +312,20 @@ _This section is non-normative_
 Clients are ephemeral, client registration is optional, and most clients cannot keep secrets. These,
 among other factors, are what makes client trust challenging.
 
-# Privacy considerations
+# Privacy Considerations
 
 _This section is non-normative_
 
-## Access Token Reuse
+## DPoP-bound Access Token Reuse
 
-With JWTs being extendable by design, there is potential for a privacy breach if Access Tokens get
-reused across multiple resource servers. It is not unimaginable that a custom claim is added to the
-Access Token on instantiation. This addition may unintentionally give other resource servers
-consuming the Access Token information about the user that they may not wish to share outside of the
-intended RS.
+With JWTs being extendable by design, there is potential for a privacy breach
+if DPoP-bound Access Tokens get reused across multiple resource servers. It is
+not unimaginable that a custom claim is added to the DPoP-bound Access Token
+on instantiation. This addition may unintentionally give other resource
+servers consuming the DPoP-bound Access Token information about the user that
+they may not wish to share outside of the intended RS.
 
-# Acknowledgments
+# Acknowledgements
```

Comment:
The American English version does not include the 'e'. See https://www.merriam-webster.com/dictionary/acknowledgment

Reply:
Am aware but my brain must have autocorrected that. I've reverted to U.S. English ( https://github.com/solid/authentication-panel/commit/c2b8798f56701da144110eb173545793da253037 ) for consistency in the document.

Personally, I find Acknowledge -> Acknowledgements to be clear, and the phonetic pronunciation looks awkward without the 'e'.

At the risk of opening up a spelling-war - but not a hill that I want to die on - I think it'd be great to use en-gb or en-ca.

Reply:
The WC3 style guide specifies en-us must be used. As an Englishman, I prefer en-gb, too. But it's cut and dry, I'm afraid.

Reply:
Note that it is a *guideline* for W3C publications ( see list https://www.w3.org/pubrules/doc ) and as far as I know, it doesn't apply to CG's reports (specs). If you and co-editors prefer to use en-us, that's within your right/call.

Reply:
I was under the understanding that that is what we were producing for the panel, a first public working draft? I'm new to W3C community groups, so if I have my terminology crossed, please correct me.

Reply:
I think you might want to see https://gitter.im/solid/authentication-panel?at=5f16bdbca4b25c73c1de01b3 and https://gitter.im/solid/authentication-panel?at=5f44d9b959ac794e02c5137c

I can try to fill you in other stuff if you like. Ping in chat (Gitter or panel meeting).